app/assets/v2/js/cart.js

```diff
@@ -1003,148 +1000,204 @@ Vue.component('grants-cart', {
         });
     },
 
-    // POSTs donation data to database
+    // POSTs donation data to database. Wrapped in a try/catch, and if it fails, we fallback to the manual ingestion script
     async postToDatabase(txHash, contractAddress, userAddress) {
-      // this.grantsByTenant is the array used for donations
-      // We loop through each donation to configure the payload then POST the required data
-      const donations = this.donationInputs;
-      const csrfmiddlewaretoken = document.querySelector('[name=csrfmiddlewaretoken]').value;
-
-      // If txHash has a length of one, stretch it so there's one hash for each donation
-      let txHashes = txHash;
-
-      if (txHash.length === 1) {
-        txHashes = new Array(donations.length).fill(txHash[0]);
-      }
+      try {
+        // this.grantsByTenant is the array used for donations
```

Comment:
Nit, just to clarify this comment
```suggestion
        // this.grantsByTenant is the array used for donations, and this.donationInputs is computed from it
```